setup.py

```diff
@@ -29,6 +28,8 @@
         'Programming Language :: Python :: 2',
         'Programming Language :: Python :: 2.7',
         'Programming Language :: Python :: 3',
+        'Programming Language :: Python :: 3.3',
+        'Programming Language :: Python :: 3.4',
```

Comment:
@iham I would rather not test these. Latest 2.x and latest 3.x would be enough. Would you mind removing them and adding 3.6?